src/hooks/useSwapCurrencyList.ts

```diff
@@ -171,12 +171,16 @@ const useSwapCurrencyList = (
           break;
         case 'highLiquidityAssets':
           setHighLiquidityAssets(
-            await searchCurrencyList(assetType, searchQuery, chainId)
+            handleVerifiedResponse(
```

Comment:
can we rename this to "handleSearchResponse" since it's no longer specific to verified tokens?